mass_editing/tests/test_mass_editing.py

```diff
@@ -38,33 +53,52 @@ def _create_partner(self):
             'email': 'example@yourcompany.com',
             'phone': 123456,
             'category_id': [(6, 0, categ_ids)],
+            'notify_email': 'always'
+        })
+
+    def _create_partner_title(self):
+        """Create a Partner Title."""
+
```

Comment:
Remove all the empty lines

Reply:
Again, I'm for them, to improve legibility. Any reason behind this suggestion?